### PR TITLE
Support OS Deployment Plan

### PR DESCRIPTION
**Pull request by AsisBagga**
### Description
Support To Os Deployment Plan 

### Issues Resolved
N/A

### Check List
- [x] New functionality includes sanity testing.
  - [x] All sanity tests pass. (`$ ansible-test sanity`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).

### CODE REVIEW

File: plugins/modules/oneview_os_deployment_plan_facts.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+###
+# Copyright (2016-2017) Hewlett Packard Enterprise Development LP
```
Comment (yuvirani) [inline plugins/modules/oneview_os_deployment_plan_facts.py:4]: Change copyright to 2020

File: plugins/modules/oneview_os_deployment_plan_facts.py [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+DOCUMENTATION = '''
+---
+module: oneview_os_deployment_plan_facts
+short_description: Retrieve facts about one or more OS Deployment Plans.
+description:
+    - Retrieve facts about one or more of the OS Deployment Plans from OneView.
+version_added: "2.3.0"
+requirements:
+    - "python >= 2.7.9"
+    - "hpeOneView >= 5.8.0"
```
Comment (yuvirani) [inline plugins/modules/oneview_os_deployment_plan_facts.py:34]: hpeOneview >= 5.4.0

File: roles/oneview_os_deployment_plan_facts/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+## oneview_os_deployment_plan_facts
+Retrieve the facts about one or more of the OneView OS Deployment Plan Facts
```
Comment (yuvirani) [inline roles/oneview_os_deployment_plan_facts/README.md:2]: Remove Facts at the end

File: tests/unit/test_oneview_os_deployment_plan_facts.py [hunk 4]
```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+###
+# Copyright (2016-2017) Hewlett Packard Enterprise Development LP
```
Comment (yuvirani) [inline tests/unit/test_oneview_os_deployment_plan_facts.py:4]: change copy right

File: plugins/modules/oneview_os_deployment_plan_facts.py (hunk 1 above)
Comment (AsisBagga) [reply plugins/modules/oneview_os_deployment_plan_facts.py:4]: done

File: plugins/modules/oneview_os_deployment_plan_facts.py (hunk 2 above)
Comment (AsisBagga) [reply plugins/modules/oneview_os_deployment_plan_facts.py:34]: done

File: tests/unit/test_oneview_os_deployment_plan_facts.py (hunk 4 above)
Comment (AsisBagga) [reply tests/unit/test_oneview_os_deployment_plan_facts.py:4]: done

File: roles/oneview_os_deployment_plan_facts/README.md (hunk 3 above)
Comment (AsisBagga) [reply roles/oneview_os_deployment_plan_facts/README.md:2]: this Read Me is about oneview_os_deployment_plan_facts hence kept it.

**Pull request merged**